src/workspaces.c

```diff
@@ -260,7 +292,17 @@ _osd_show(struct server *server)
 void
 workspaces_init(struct server *server)
 {
-	wl_list_init(&server->workspaces);
+	server->workspaces.cosmic_manager = wlr_cosmic_workspace_manager_create(
+		server->wl_display, CW_CAP_WS_ACTIVATE | CW_CAP_GRP_WS_CREATE, 1);
```

Comment:
In `workspaces_init()` using `1` as last argument in `wlr_cosmic_workspace_manager_create()` is not easy to read. Suggest a `#define` (is it COSMIC_WORKSPACE_V1_VERSION?)

Reply:
added